Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -1239,5 +1239,40 @@ public async Task<string> PresignedPutObjectAsync(string bucketName, string obje
 
             return Tuple.Create(this.restClient.BaseUrl.AbsoluteUri, policy.GetFormData());
         }
+
+
+        /// <summary>
+        /// Get the configuration object for Legal Hold Status 
+        /// </summary>
+        /// <param name="args">GetObjectLegalHoldArgs Arguments Object which has object identifier information - bucket name, region, object name, version ID</param>
+        /// <param name="cancellationToken">Optional cancellation token to cancel the operation </param>
+        /// <returns> Legal Hold configuration in ObjectLegalHoldConfiguration object  </returns>
+        /// <exception cref="InvalidBucketNameException">When bucketName is invalid</exception>
+        /// <exception cref="InvalidObjectNameException">When bucketName is invalid</exception>
+
+        public async Task<ObjectLegalHoldConfiguration> GetLegalHoldConfigurationAsync(GetObjectLegalHoldArgs args, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
the return can be a bool. Please check if legalhold is not set, if other SDKs throw an exception.

Reply:
Ideally, I would think `off` if it has never been set and user permission to read legalholdstatus

Reply:
Will make return to bool. If LegalHold is not set, UnexpectedMinioException exception is thrown with "The specified object does not have a ObjectLock configuration"